serializer.rst

```diff
@@ -148,6 +148,29 @@ properties and setters (``setXxx()``) to change properties:
             ;
         };
 
+Serializer context
+------------------
+
+The serializer can use a (de)serialization context to control how a resource is (de)serialized.
+The context is passed to all normalizers. For example:
+
+* :class:`Symfony\\Component\\Serializer\\Normalizer\\DateTimeNormalizer` uses ``datetime_format`` key as date time format;
+* :class:`Symfony\\Component\\Serializer\\Normalizer\\AbstractObjectNormalizer` uses ``empty_iterable_as_object`` to preserve empty objects (keeps ``{}`` instead of ``[]`` in JSON);
+
+.. versionadded:: 5.3
```

Comment:
```suggestion
.. versionadded:: 5.4
```
according to the docs issue you want to close with this PR

Reply:
Indeed, thank